messaging/src/main/java/org/axonframework/queryhandling/duplication/DuplicateQueryHandlerResolution.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.queryhandling.duplication;
```

Comment:
I think a package like `subscribing`, `registration`, or something alike, is more reasonable. 
Because this is part of the handler registration process.

Reply:
Renamed to `registration`